### PR TITLE
Improved animations, added explanations, compacted some code

### PR DESCRIPTION
**Pull request by mrcendre**
This adds few line on KGStatusBar/KGStatusBar.m to improve the animations of the stringLabel object.
It also adds some informations about the different commands' behavior.
Finally, some useless default lines in KGStatusBarExample/AppDelegate.m to lighten the final code.
That's it!

### CODE REVIEW

File: KGStatusBar/KGStatusBar.m [hunk 1]
```diff
@@ -87,12 +89,14 @@ - (void) dismiss
 {
     [UIView animateWithDuration:0.4 animations:^{
         self.stringLabel.alpha = 0.0;
+        self.stringLabel.frame = CGRectMake(self.stringLabel.frame.origin.x, self.stringLabel.frame.origin.y + 7, self.stringLabel.frame.size.width, self.stringLabel.frame.size.height);
     } completion:^(BOOL finished) {
+        [UIView animateWithDuration:0.3 animations:^{topBar.alpha = 0;} completion:^(BOOL finishedd) {
```
Comment (mrcendre) [inline KGStatusBar/KGStatusBar.m:94]: Fade out the black status bar, while the last version dismisses it directly.

File: KGStatusBar/KGStatusBar.m [hunk 2]
```diff
@@ -72,13 +73,14 @@ - (void)showWithStatus:(NSString *)status barColor:(UIColor*)barColor textColor:
         
         labelRect = CGRectMake((self.topBar.frame.size.width / 2) - (stringWidth / 2), 0, stringWidth, stringHeight);
     }
-    self.stringLabel.frame = labelRect;
+    [self.stringLabel setFrame:CGRectMake(labelRect.origin.x, labelRect.origin.y + 7, labelRect.size.width, labelRect.size.height)];
```
Comment (mrcendre) [inline KGStatusBar/KGStatusBar.m:76]: Along with the line 83, these two lines adds another animation: the label goes up on showing and goes down on dismiss.

File: KGStatusBarExample/KGStatusBarExample/KGViewController.m [hunk 3]
```diff
@@ -15,36 +15,27 @@ - (IBAction)errorButtonPressed:(id)sender;
...
 - (IBAction)errorButtonPressed:(id)sender {
+    //Temporary message too
     [KGStatusBar showErrorWithStatus:@"Error syncing files"];
 }
-
 - (IBAction)statusButtonPressed:(id)sender {
+    //A persistent message : will not dismiss until -(IBAction)dismissButtonPressed:(id)sender is called.
     [KGStatusBar showWithStatus:@"Loading..."];
 }
-
```
Comment (mrcendre) [inline KGStatusBarExample/KGStatusBarExample/KGViewController.m:46]: All the commented line at the bottom of this one (except the 26th) give informations about the status bar behavior.

**Comment by kevingibbon** [comment]
Sorry dude. I really don't like how it pops in from the top.

**Pull request closed without merging**